ui/src/components/home/VisionMissionSection.js

```diff
@@ -0,0 +1,412 @@
+import {
+  Button,
+  Box,
+  Flex,
+  HStack,
+  Heading,
+  VStack,
+  Text,
+  Spacer,
+  Link,
+  Image,
+  Fade,
+  Container,
+} from '@chakra-ui/react';
+import { ArrowForwardIcon } from '@chakra-ui/icons';
+import { InView } from 'react-intersection-observer';
+const VisionMissionSection = () => {
+  return (
+    <InView rootMargin="-200px" triggerOnce={true}>
+      {({ inView, ref }) => (
+        <Fade transition={{ enter: { duration: 1 } }} in={inView}>
+          <Container
+            maxW="container.xl"
+            py={10}
+            ref={ref}
+            bgImage={`url('${process.env.PUBLIC_URL}/images/home/hk-green.png')`}
+            bgPos="center right"
+            bgSize="contain"
+            bgRepeat="no-repeat"
+          >
+            <VStack>
+              <Box w="100%" alignItems="flex-start" paddingBottom={'1em'}>
+                <Box w="20%" position="relative">
+                  <Box
+                    w="100%"
+                    position="absolute"
+                    h="100%"
+                    bgPos="45% 50%"
+                    bgSize="40%"
+                    bgRepeat="no-repeat"
+                    bgImage={`url('${process.env.PUBLIC_URL}/images/home/vector-green-1.svg')`}
+                  ></Box>
+                  <Box
+                    w="100%"
+                    position="absolute"
+                    h="107%"
+                    bgPos="30% 100%"
+                    bgSize="50%"
+                    bgRepeat="no-repeat"
+                    bgImage={`url('${process.env.PUBLIC_URL}/images/home/vector-green-2.svg')`}
+                  ></Box>
+                  <Heading
+                    color="#0C0C20"
+                    fontFamily="DMSerifDisplay_Italic, serif"
+                    fontWeight="400"
+                    lineHeight="1"
+                    wordBreak="break-word"
+                  >
+                    Our Vision<br></br> & Mission
+                  </Heading>
+                </Box>
+              </Box>
+              <Flex>
+                <VStack
+                  borderRadius="0.5rem"
```

Comment:
for paddings and border radius, using the built in chakra reactive numbers is usually more reactive towards ui change.

i.e. `borderRadius="10"`

Also the padding and border looks different from design.

Instead of `1em` I think you can do `p=10`

Reply:
<img width="181" alt="image" src="https://github.com/user-attachments/assets/697ad3be-d681-4097-8b63-23020fb37a94">
design padding is a lot bigger

<img width="211" alt="image" src="https://github.com/user-attachments/assets/e4cd5aa7-e305-416e-982e-36f11ddb19df">


similar problem for most paddings